public/index.html

```diff
@@ -2,6 +2,7 @@
 <head>
   <link rel="stylesheet" href="styles.css" type="text/css">
   <link rel="stylesheet" href="https://cdn.rawgit.com/twbs/bootstrap/48938155eb24b4ccdde09426066869504c6dab3c/dist/css/bootstrap.min.css">
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDYeSF4j651kM9Zyc0Agk2ABDHad1HF2U4"></script>
```

Comment:
@tgrahams101 : Is this your Google key? If so, you probably shouldn't put it in the repo even if it's a private repo.